src/chttpd/src/chttpd_db.erl

```diff
@@ -79,6 +79,8 @@ handle_request(#httpd{path_parts=[DbName|RestParts],method=Method}=Req)->
          end;
     {_, []} ->
         do_db_req(Req, fun db_req/2);
+    {_, [<<"_restore">>|_]} ->
```

Comment:
This needs to go into the handlers list [1] and not be hard coded at this level.

[1] https://github.com/apache/couchdb/blob/prototype/fdb-layer/src/chttpd/src/chttpd_httpd_handlers.erl#L20-L33

Reply:
Also, `_undelete` might be a good replacement given the mailing list freedback.